test/easyconfigs/easyconfigs.py

```diff
@@ -1428,22 +1428,17 @@ def test_pr_https(self):
         """Make sure https:// URL is used (if it exists) for homepage/source_urls (rather than http://)."""
 
         whitelist = [
-            'Kaiju',  # invalid certificate at https://kaiju.binf.ku.dk
-            'p4vasp',  # https://www.p4vasp.at doesn't work
             'ITSTool',  # https://itstool.org/ doesn't work
-            'UCX-',  # bad certificate for https://www.openucx.org
-            'MUMPS',  # https://mumps.enseeiht.fr doesn't work
-            'PyFR',  # https://www.pyfr.org doesn't work
+            'libxml2',  # https://xmlsoft.org works, but invalid certificate
```

Comment:
and do we need to keep this around, given that the site is dead/no longer the official url

Reply:
Removed. I meant to remove this in the PR that updated the urls.